src/raftstore/store/fsm/mod.rs

```diff
@@ -57,58 +57,89 @@ use import::SSTImporter;
 type Key = Vec<u8>;
 
 pub struct Store<T, C: 'static> {
+    /// store_tag, "[store <store_id>]"
+    tag: String,
+    /// the store meta information of the store
```

Comment:
```suggestion
    /// The store meta information of the store.
```

Reply:
it is not a sentence, should we use the capital form?

Reply:
@Connor1996 Yes. Please capitalize the first letter and leave a period for both phrases and sentences. [Rust ref link](https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#exporting-a-convenient-public-api-with-pub-use).